tools/maplint/README.md

```diff
@@ -122,6 +122,19 @@ Similar to [banned_neighbors](#banned_neighbors), you can specify a regular expr
       allow: { pattern: '^[A-Z].*$' }
 ```
 
+### `ignore`
+If you want to exclude other objects from `banned_neighbors`, you can specify `ignore`.
+
+This is just a list of types that will get skipped. This lint will skip banning the kitten bowl subtypes from being on the same tile as a dog bowl.
```

Comment:
Nit: I'd rewrite this to say what this does and doesn't catch, because I'm a little confused

Reply:
This should clear it up https://github.com/tgstation/tgstation/pull/91090/commits/16c9988fbf5bebd7a420f553bfb1dfbee45e3f04